File: year2/systems-design-implementation/Lab7/SwimModel/src/main/java/domain/Proba.java
package domain;

import utils.Entity;

import java.io.Serializable;

public class Proba extends Entity<Integer> implements Serializable {

    private Integer distanta;
    private String stil;

    public Proba(Integer id, Integer distanta, String stil) {
        super(id);
        this.distanta = distanta;
        this.stil = stil;
    }

    public Integer getDistanta() {
        return distanta;
    }

    public void setDistanta(Integer distanta) {
        this.distanta = distanta;
    }

    public String getStil() {
        return stil;
    }

    public void setStil(String stil) {
        this.stil = stil;
    }

    @Override
    public String toString() {
        return stil + " " + distanta + "m";
    }
}

File: year2/systems-design-implementation/Lab2/Lab2Sharp/Domain/Proba.cs
using Lab2Sharp.Utils;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab2Sharp.Domain
{
    class Proba : Entity<int>
    {
        public String stil { get; set; }
        public int distanta { get; set; }

        public override string ToString()
        {
            return base.ToString() + "proba " + stil + ", " + distanta + "m";
        }
    }
}
